### PR TITLE
TG-1764 create host as a part of schedule event and rename constants to constant and utilities to utility

### PR DESCRIPTION
**Pull request by ankit45621**
create host and rename `constants` folder to `constant` and `utilities` folder to `utilitie`

### CODE REVIEW

File: src/main/java/com/abcfinancial/data/generator/CreateHostDataGeneratorTest.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CreateHostDataGeneratorTest {
+
+    private static final Logger logger = LoggerFactory.getLogger(CreateHostDataGeneratorTest.class);
+
+    public static void main(String[] args) {
+
+        logger.info("***** Starting Create Event Data Generator Test Started *****");
```
Comment (sumitabc) [inline src/main/java/com/abcfinancial/data/generator/CreateHostDataGeneratorTest.java]: change it to create Host Data Generator Test Started

File: src/main/java/com/abcfinancial/data/generator/CreateHostDataGeneratorTest.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        GenerateData.generateData(AppConstant.CREATE_HOST);
+        logger.info("***** Starting Starting Create Event Data Generator Test Ended*****");
```
Comment (sumitabc) [inline src/main/java/com/abcfinancial/data/generator/CreateHostDataGeneratorTest.java]: same here as above

File: src/main/java/com/abcfinancial/data/service/GenerateData.java [hunk 3]
```diff
@@ -13,8 +13,12 @@
 import java.util.Map;
 
 public class GenerateData {
+
+    private GenerateData() {
+    }
+
     private static final Logger logger = LoggerFactory.getLogger(GenerateData.class);
-    public  static final Map<String, String> fileMap = new HashMap<String, String>();
+    protected  static final Map<String, String> fileMap = new HashMap<>();
```
Comment (sumitabc) [inline src/main/java/com/abcfinancial/data/service/GenerateData.java]: logic behind using protected?

File: src/main/java/com/abcfinancial/data/constant/AppConstant.java [hunk 4]
```diff
@@ -24,8 +26,8 @@
     public static final String PACKAGE = "package";
     public static final String UNLIMITED = "unlimited";
```
Comment (magarwal121) [inline src/main/java/com/abcfinancial/data/constant/AppConstant.java]: we can use Boolean variable as UNLIMITED= true or false

File: src/main/java/com/abcfinancial/data/service/GenerateData.java [hunk 5]
```diff
@@ -12,29 +12,38 @@
...
-        String isLimited=appConfig.getString(AppConstant.UNLIMITED);
-        List configList=null;
-        if("true".equals(isLimited)){
-             configList = appConfig.getList(AppConstant.CONFIG_LIST_NO_QUANTITY);
+        fileMap.put( "fileName", configName );
+        String extension = appConfig.getString( AppConstant.EXTENSION );
+        int recordCount = appConfig.getInt( AppConstant.COUNT );
+        String isLimited = appConfig.getString( AppConstant.UNLIMITED );
+        List configList = null;
+        if( "true".equals( isLimited ) )
```
Comment (magarwal121) [inline src/main/java/com/abcfinancial/data/service/GenerateData.java]: use as Boolean not string

Comment (ankit45621) [reply src/main/java/com/abcfinancial/data/service/GenerateData.java]: boolean use done.

Comment (ankit45621) [reply src/main/java/com/abcfinancial/data/service/GenerateData.java]: Done

**Pull request merged**